diff -r -U 3 minecraft/net/minecraft/src/CodecMus.java minecraft_patched/net/minecraft/src/CodecMus.java
--- minecraft/net/minecraft/src/CodecMus.java	2019-11-22 07:58:44.736441900 +0700
+++ minecraft_patched/net/minecraft/src/CodecMus.java	2019-11-22 08:01:52.144163400 +0700
@@ -1,10 +1,15 @@
 package net.minecraft.src;
 
+import java.io.IOException;
 import java.io.InputStream;
 import paulscode.sound.codecs.CodecJOrbis;
 
 public class CodecMus extends CodecJOrbis {
     protected InputStream openInputStream() {
-        return new MusInputStream(this, this.url, this.urlConnection.getInputStream());
+        try {
+            return new MusInputStream(this, this.url, this.urlConnection.getInputStream());
+        } catch (IOException ignore) {
+            return null;
+        }
     }
 }
diff -r -U 3 minecraft/net/minecraft/src/EntityCreeper.java minecraft_patched/net/minecraft/src/EntityCreeper.java
--- minecraft/net/minecraft/src/EntityCreeper.java	2019-11-28 18:52:35.523776600 +0700
+++ minecraft_patched/net/minecraft/src/EntityCreeper.java	2019-11-28 18:53:10.578914300 +0700
@@ -11,7 +11,7 @@
 
     protected void func_21057_b() {
         super.func_21057_b();
-        this.field_21064_bx.func_21124_a(16, -1);
+        this.field_21064_bx.func_21124_a(16, (byte)-1);
         this.field_21064_bx.func_21124_a(17, (byte)0);
     }
 

diff -r -U 3 minecraft/net/minecraft/src/MusInputStream.java minecraft_patched/net/minecraft/src/MusInputStream.java
--- minecraft/net/minecraft/src/MusInputStream.java	2019-11-22 07:58:45.075254200 +0700
+++ minecraft_patched/net/minecraft/src/MusInputStream.java	2019-11-22 08:03:03.611201500 +0700
@@ -1,5 +1,6 @@
 package net.minecraft.src;
 
+import java.io.IOException;
 import java.io.InputStream;
 import java.net.URL;
 
@@ -25,7 +26,11 @@
     }
 
     public int read(byte[] var1, int var2, int var3) {
-        var3 = this.field_1515_d.read(var1, var2, var3);
+        try {
+            var3 = this.field_1515_d.read(var1, var2, var3);
+        } catch (IOException ignore) {
+            return 0;
+        }
 
         for(int var4 = 0; var4 < var3; ++var4) {
             byte var5 = var1[var2 + var4] = (byte)(var1[var2 + var4] ^ this.field_1516_c >> 8);
