diff -r -U 3 minecraft_server/net/minecraft/src/EnumCreatureType.java minecraft_server_patched/net/minecraft/src/EnumCreatureType.java
--- minecraft_server/net/minecraft/src/EnumCreatureType.java	2020-05-20 01:08:29.419072000 +0700
+++ minecraft_server_patched/net/minecraft/src/EnumCreatureType.java	2020-05-20 01:30:06.123036500 +0700
@@ -6,9 +6,9 @@
 import net.minecraft.src.Material;
 
 public enum EnumCreatureType {
-   monster,
-   creature,
-   waterCreature;
+   monster(IMob.class, 70, Material.field_526_a, false),
+   creature(EntityAnimal.class, 15, Material.field_526_a, true),
+   waterCreature(EntityWaterMob.class, 5, Material.field_521_f, true);
 
    private final Class field_21109_d;
    private final int field_21108_e;
@@ -37,10 +37,4 @@
    public boolean func_21103_d() {
       return this.field_21106_g;
    }
-
-   static {
-      monster = new EnumCreatureType("monster", 0, IMob.class, 70, Material.field_526_a, false);
-      creature = new EnumCreatureType("creature", 1, EntityAnimal.class, 15, Material.field_526_a, true);
-      waterCreature = new EnumCreatureType("waterCreature", 2, EntityWaterMob.class, 5, Material.field_521_f, true);
-   }
 }
