diff -r -U 3 minecraft/net/minecraft/src/EntityCreeper.java minecraft_patched/net/minecraft/src/EntityCreeper.java
--- minecraft/net/minecraft/src/EntityCreeper.java	2019-11-28 18:52:35.523776600 +0700
+++ minecraft_patched/net/minecraft/src/EntityCreeper.java	2019-11-28 18:53:10.578914300 +0700
@@ -11,7 +11,7 @@
 
     protected void func_21044_a() {
         super.func_21044_a();
-        this.field_21045_af.func_21153_a(16, -1);
+        this.field_21045_af.func_21153_a(16, (byte)-1);
         this.field_21045_af.func_21153_a(17, (byte)0);
     }
 